app/views/articles/index.html.haml

```diff
@@ -1,18 +1,29 @@
 %h1 List of articles
+= render 'search', q: @q
```

Comment:
but this enables to try different search forms by not losing old implementations